### PR TITLE
fix some clippy warnings

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by est31** [comment]
Thanks! the try -> ? stuff is okay but I dont like the needless return lint of clippy. explicit return statements are fine.

File: src/reading.rs [hunk 1]
```diff
@@ -241,7 +241,7 @@ impl PageParser {
 			page_siz += *val as u16;
 			// Increment by 1 if val < 255, otherwise by 0
 			self.packet_count += (*val < 255) as u16;
-			self.bi.ends_with_continued = !(*val < 255);
```
Comment (est31) [inline src/reading.rs]: This piece of code used to be analog to the code above. I think it's better in the old state.

**Review by est31 — CHANGES_REQUESTED** [review]

File: src/reading.rs [hunk 2]
```diff
@@ -1114,8 +1114,8 @@ pub mod async_api {
 	}
 
 	/**
-	Async packet reading functionality.
```
Comment (est31) [inline src/reading.rs]: Why are you replacing tabs with spaces here?

Comment (ghost) [reply src/reading.rs]: Clippy tell it is better but i can revert it if you prefer.

File: src/reading.rs (hunk 1 above)
Comment (ghost) [reply src/reading.rs]: revert.

**Review by est31 — APPROVED** [review]
Thanks!

**Comment by est31** [comment]
There is a CI failure. I think you need to allow unknown ~warnings~ *lints*.

**Review by est31 — APPROVED** [review]

**Comment by est31** [comment]
Huh it's still failing with the same error. Maybe try reordering the allow to be higher?

**Pull request merged**